neofetch

```diff
@@ -242,7 +242,13 @@ get_model() {
             fi
         ;;
 
-        "Mac OS X") model="$(sysctl -n hw.model)" ;;
+        "Mac OS X") 
+            if [[ $(kextstat | grep "FakeSMC") != "" ]]; then
```

Comment:
Change this line to `if [[ "$(kextstat | grep FakeSMC") ]]; then` and I'll merge this. We enclose all `$()` statements in quotes and the testing for `!= ""` isn't needed as the if statement will fail and go to else if the line is blank anyway.